run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem

```diff
@@ -61,7 +61,8 @@ VerboseOnCores:              root       # Accepted values: root all
     --> NEI2016_MONMEAN        :       false    # 2002-2020
     --> DICE_Africa            :       false    # 2013
 # ----- GLOBAL INVENTORIES ----------------------------------------------------
-    --> CEDSv2                 :       ${RUNDIR_USE_CEDS}    # 1750-2019
+    --> CEDS_01x01             :       ${RUNDIR_USE_CEDS}    # 1980-2019
+    --> CEDSv2                 :       false    # 1750-2019
     --> CEDS_GBDMAPS           :       false    # 1970-2017
     --> CEDS_GBDMAPS_byFuelType:       false    # 1970-2017
```

Comment:
It may be confusing to users to have four CEDS options available. Could we retire the `CEDSv2` and/or CEDS_GBDMAPS` options from the default HEMCO_Config.rc? They could always be manually added in by users who want to use those inventories.

Tagging @1Dandan and @randallvmartin for thoughts.